packages/lib/semaphore-identity-v3-wrapper/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@pcd/semaphore-identity-v3-wrapper",
+  "private": false,
+  "version": "0.0.1",
+  "license": "GPL-3.0-or-later",
```

Comment:
Semaphore uses an MIT license (in both v3 and v4) so this package should too.

Reply:
Why? I can't think of a reason why this might be the case.

Reply:
To be more precise: GPL is what our template package uses, so all newly-created packages get that as a default. Almost all of our packages have GPL LICENSE files, as does the repository root. Nothing in the MIT license says that it can't be depended on by a GPL-licensed package. Since GPL is the default choice and, without some understanding of what the consequences would be, I am not sure that changing to a different license is something I'd feel entitled to do.

Reply:
An additional explanation about my preference here.  It is always legal for us to change an MIT-licensed package to GPL, since it's strictly more restrictive.  It's not legal to go in the other direction on arbitrary code with arbitrary contributors (it can be for code you hold full ownership of - but that's a special case).  So the best-practice I'm suggesting here is about maximizing future options, and not making one-way decisions.

I won't die on this hill for this package, which isn't depended on by @pcd/pod, but there is a reasoning here I thought was worth sharing.

Reply:
I don't think anyone has to die on a hill, I was just under the impression that GPL was our primary and default license, and that we would need good reasons for deviating from it. If that's not the case then perhaps the template package ought to be updated.

Either way, this package can happily be MIT-licensed, and I can't imagine any problem arising from making that change, especially for something as trivial.